public/app/plugins/datasource/grafana-azure-monitor-datasource/resourcePicker/resourcePickerData.ts

```diff
@@ -312,4 +319,12 @@ export default class ResourcePickerData extends DataSourceWithBackend<AzureMonit
       throw error;
     }
   }
+
+  private filterByType = (t: ResourcePickerQueryType) => {
+    const logsSupportedResourceTypesKusto = logsResourceTypes.map((v) => `"${v}"`).join(',');
+    const supportedMetricNamespacesKusto = supportedMetricNamespaces.map((v) => `"${v.toLocaleLowerCase()}"`).join(',');
+    return t === 'logs'
```

Comment:
Two unnecessary loops here but I guess the increased readability makes it worth it.

Reply:
good point, it's also recalculating it everytime the function is called. It's not a super heavy thing but can be improved